server/routes/nl/helpers.py

```diff
@@ -208,6 +208,10 @@ def parse_query_and_detect(request: Dict, backend: str, client: str,
     return None, err_json
   counters.timeit('query_detection', start)
 
+  allow_non_place = False
```

Comment:
nit: allow_entity, so that further down it reads better (`if not allow_non_place:`)

Reply:
done